src/rms/queries/LoginQueries.java

```diff
@@ -27,6 +27,30 @@ public Users loginOnUserName(String userName, String password) throws EmptyResul
 		return jtemp.queryForObject("SELECT * FROM USERS WHERE user_name = ? AND user_password = ?", 
 				new UsersMapper(), userName, password);
 	}
-
 	
+	public boolean checkIsAdminUsingEmail(String email, String password) throws EmptyResultDataAccessException {
+		int userType = jtemp.queryForObject("SELECT user_type FROM users WHERE user_email = ? AND user_password = ?", 
+				int.class, email, password);
+		if(userType == 1 || userType == 3)
```

Comment:
You should replace the if/else statement with just "return userType == 1 || userType == 3"

Reply:
That works too, but I personally prefer readability over compactness.